Makefile

```diff
@@ -5,22 +5,27 @@ SPARQL=sparql
 ROBOT_ENV=ROBOT_JAVA_ARGS=-Xmx12G
 ROBOT=$(ROBOT_ENV) robot
 
-all: $(BUILD_DIR)/phenoscape-ontology-classified.ofn
+all: $(BUILD_DIR)/phenoscape-kb.ttl $(BUILD_DIR)/phenoscape-kb-tbox-hierarchy.ofn $(BUILD_DIR)/qualities.txt $(BUILD_DIR)/anatomical_entities.txt
 
 clean:
 	rm -rf $(BUILD_DIR)
 
+# Create build directory
+$(BUILD_DIR): clean
+	mkdir -p $@
+
 # Ontologies.ofn - list of ontologies to be imported
 # Mirror ontologies locally
 $(BUILD_DIR)/mirror: ontologies.ofn
-	mkdir -p $(BUILD_DIR) && rm -rf $@ &&\
 	$(ROBOT) mirror -i $< -d $@ -o $@/catalog-v001.xml
 
 # Extract ontology metadata
 $(BUILD_DIR)/ontology-metadata: ontologies.ofn $(SPARQL)/ontology-versions.sparql
-	mkdir -p (BUILD_DIR)/ontology-metadata \
-	$(ROBOT) query -i $< --use-graphs true --queries $(SPARQL)/ontology-versions.sparql --output-dir $@
+	mkdir -p $@ \
+	$(ROBOT) query --format ttl -i $< --use-graphs true --queries $(SPARQL)/ontology-versions.sparql --output-dir $@ \
 
+$(BUILD_DIR)/ontology-versions.ttl: $(BUILD_DIR)/ontology-metadata
```

Comment:
Can't this be just one query? I don't think we need two targets, do we? You can query all the ontologies simultaneously.

Reply:
Yeah, but from what I understood, `--queries` option will generate separate output file for each input file, whereas we need a single `.ttl` file containing all the ontology metadata to be merged into `phenoscape-kb.ttl`. I can directly merge all these separate metadata files into the final kb directly instead of creating an intermediate merged metadata file though.